### PR TITLE
Updating translation files for metrics 3/4

### PR DESCRIPTION
**Pull request by tonyreigns**

### CODE REVIEW

**Comment by tonyreigns** [comment]
#build

**Comment by tonyreigns** [comment]
#build

**Comment by tonyreigns** [comment]
#build

**Comment by tonyreigns** [comment]
#libby

**Comment by pgunapal** [comment]
#build

**Review by Channyboy — DISMISSED** [review]
lgtm

**Comment by wraschke** [comment]
#libby #build

**Comment by wraschke** [comment]
#libby

**Comment by tonyreigns** [comment]
#build

File: dev/com.ibm.ws.microprofile.metrics.common/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops [hunk 1]
```diff
@@ -101,6 +101,21 @@ libraryRefConfigured.info.CWMMC0014I=CWMMC0014I: The MicroProfile Metrics featur
 libraryRefConfigured.info.CWMMC0014I.explanation=The MicroProfile Metrics feature is configured to use a libraryRef. This referenced library must contain the Micrometer core library, target Micrometer libraries and dependencies.
 libraryRefConfigured.info.CWMMC0014I.useraction=No action is required.
 
+#MicroProfile Metrics 3.0/4.0 configuration errors
+invalidHistogramValueConfigured.warning.CWMMC0015W=CWMMC0015W: The value {0} is invalid for the {1} property. Only integer and decimal values are accepted.
+invalidHistogramValueConfigured.warning.CWMMC0015W.explanation=The configured histogram values are invalid.
+invalidHistogramValueConfigured.warning.CWMMC0015W.useraction=Only configure the values with integers and decimals.
```
Comment (ramkumar-k-9286) [inline dev/com.ibm.ws.microprofile.metrics.common/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops]:
```suggestion
invalidHistogramValueConfigured.warning.CWMMC0015W.useraction=Configure the values with integers and decimals only.
```

File: dev/com.ibm.ws.microprofile.metrics.common/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops [hunk 2]
```diff
@@ -101,6 +101,21 @@ libraryRefConfigured.info.CWMMC0014I=CWMMC0014I: The MicroProfile Metrics featur
...
+
+#MicroProfile Metrics 3.0/4.0 configuration errors
+invalidTimerValueConfigured.warning.CWMMC0016W=CWMMC0016W: The value {0} is invalid for the {1} property. Only integer values with an optional time unit (e.g. ms,s,m,h) are accepted.
```
Comment (ramkumar-k-9286) [inline dev/com.ibm.ws.microprofile.metrics.common/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops]:
```suggestion
invalidTimerValueConfigured.warning.CWMMC0016W=CWMMC0016W: The value {0} is invalid for the {1} property. Only integer values with a time unit (for example ms,s,m,h) are accepted.
```

File: dev/com.ibm.ws.microprofile.metrics.common/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops [hunk 3]
```diff
@@ -101,6 +101,21 @@ libraryRefConfigured.info.CWMMC0014I=CWMMC0014I: The MicroProfile Metrics featur
...
+invalidTimerValueConfigured.warning.CWMMC0016W.explanation=The configured timer values are invalid.
+invalidTimerValueConfigured.warning.CWMMC0016W.useraction=Only configure the values with integer values and optional time units (e.g. ms,s,m,h
```
Comment (ramkumar-k-9286) [inline dev/com.ibm.ws.microprofile.metrics.common/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops]:
```suggestion
invalidTimerValueConfigured.warning.CWMMC0016W.useraction=Configure with integer values and time units (for example ms,s,m,h).
```

File: dev/com.ibm.ws.microprofile.metrics.common/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops [hunk 4]
```diff
@@ -101,6 +101,21 @@ libraryRefConfigured.info.CWMMC0014I=CWMMC0014I: The MicroProfile Metrics featur
...
+
+#MicroProfile Metrics 3.0/4.0 configuration errors
+invalidPercentileValueConfigured.warning.CWMMC0017W=CWMMC0017W: The value {0} is invalid for the {1} property. Only values 0.0-1.0 inclusively are accepted.
```
Comment (ramkumar-k-9286) [inline dev/com.ibm.ws.microprofile.metrics.common/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops]:
```suggestion
invalidPercentileValueConfigured.warning.CWMMC0017W=CWMMC0017W: The value {0} is invalid for the {1} property. Values must be in the range of 0.0 to 1.0 inclusive to be accepted.
```

File: dev/com.ibm.ws.microprofile.metrics.common/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops [hunk 5]
```diff
@@ -101,6 +101,21 @@ libraryRefConfigured.info.CWMMC0014I=CWMMC0014I: The MicroProfile Metrics featur
...
+invalidPercentileValueConfigured.warning.CWMMC0017W.explanation=The configured percentile values are invalid.
+invalidPercentileValueConfigured.warning.CWMMC0017W.useraction=Only configure the values between 0.0-1.0
```
Comment (ramkumar-k-9286) [inline dev/com.ibm.ws.microprofile.metrics.common/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops]:
```suggestion
invalidPercentileValueConfigured.warning.CWMMC0017W.useraction=Configure the values in the 0.0 to 1.0 inclusive range to be accepted.
```

**Comment by tonyreigns** [comment]
#build

**Comment by tonyreigns** [comment]
#build

**Review by pgunapal — APPROVED** [review]
LGTM

**Comment by tonyreigns** [comment]
#libby

**Pull request merged**